.idea/misc.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="ASMIdeaPluginConfiguration">
```

Comment:
Please, do not include irrelevant changes.

Reply:
Already discarded ^^^